src/mappings/mappingHandlers.ts

```diff
@@ -208,6 +208,72 @@ export async function handleCW20Transfer(event: CosmosEvent): Promise<void> {
   await cw20transfer.save();
 }
 
+export async function handleCW20BalanceBurn(event: CosmosEvent): Promise<void> {
+  const id = messageId(event.msg);
```

Comment:
I'm thinking we should consider appending "-mint" and "-burn" to the id's for those balance change events for consistency (as we're already doing "-credit" / "-debit" for transfers).